source/localizable/routing/specifying-a-routes-model.md

```diff
@@ -23,7 +23,7 @@ export default Ember.Route.extend({
 ```
 
 Typically, the `model` hook should return an [Ember Data](../../models/) record,
-but it can also return any [promise](https://www.promisejs.org/) object (Ember
```

Comment:
How about we say promise instead of promise object? What do you think about that @locks?